pages/interface/components/Markdown/copyToClipboardPlugin.js

```diff
@@ -0,0 +1,37 @@
+const copyIconSvg =
+  '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="20" height="20"><path d="M7.024 3.75c0-.966.784-1.75 1.75-1.75H20.25c.966 0 1.75.784 1.75 1.75v11.498a1.75 1.75 0 0 1-1.75 1.75H8.774a1.75 1.75 0 0 1-1.75-1.75Zm1.75-.25a.25.25 0 0 0-.25.25v11.498c0 .139.112.25.25.25H20.25a.25.25 0 0 0 .25-.25V3.75a.25.25 0 0 0-.25-.25Z"></path><path d="M1.995 10.749a1.75 1.75 0 0 1 1.75-1.751H5.25a.75.75 0 1 1 0 1.5H3.745a.25.25 0 0 0-.25.25L3.5 20.25c0 .138.111.25.25.25h9.5a.25.25 0 0 0 .25-.25v-1.51a.75.75 0 1 1 1.5 0v1.51A1.75 1.75 0 0 1 13.25 22h-9.5A1.75 1.75 0 0 1 2 20.25l-.005-9.501Z"></path></svg>';
+
+const checkIconSvg =
+  '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="20" height="20"><path d="M21.03 5.72a.75.75 0 0 1 0 1.06l-11.5 11.5a.747.747 0 0 1-1.072-.012l-5.5-5.75a.75.75 0 1 1 1.084-1.036l4.97 5.195L19.97 5.72a.75.75 0 0 1 1.06 0Z"></path></svg>';
+
+function copyToClipboard(text, buttonElement) {
+  navigator.clipboard.writeText(text).then(() => {
+    buttonElement.innerHTML = checkIconSvg;
+    buttonElement.title = 'Copiado!';
+    buttonElement.classList.add('copied');
+
+    setTimeout(() => {
+      buttonElement.innerHTML = copyIconSvg;
+      buttonElement.title = 'Copiar';
+      buttonElement.classList.remove('copied');
+    }, 2500);
+  });
+}
```

Comment:
```suggestion
import octicons from '@primer/octicons';

function copyToClipboard(text, buttonElement) {
  navigator.clipboard.writeText(text).then(() => {
    buttonElement.innerHTML = octicons.check.toSVG();
    buttonElement.title = 'Copiado!';
    buttonElement.classList.add('copied');

    setTimeout(() => {
      buttonElement.innerHTML = octicons.copy.toSVG();
      buttonElement.title = 'Copiar';
      buttonElement.classList.remove('copied');
    }, 2500);
  });
}
```

Não são sugestões, mas sim questionamentos mesmo...

Será que não deveríamos utilizar a biblioteca ao invés de apenas copiar os ícones e deixar fixos no código? Obs. Eu não testei se o código está correto, apenas me baseei na [documentação](https://github.com/primer/octicons/tree/main/lib/octicons_node#octiconsnametosvg).

Caso seja melhor deixar fixo no código, não seria interessante informar de onde vieram?

Será que ficaria melhor deixar esse arquivo dentro de uma pasta como `markdown>plugins`?